### PR TITLE
Inherited Permissions

### PR DESCRIPTION
**Pull request by mikemand**
This pull request is a work in progress. There are a couple things (that I can think of, and probably a hundred more that I cannot) that need to be completed before this is ready to merge, but I wanted to get this in here now since I've already put a good amount of thought into it.

Todo:
- [ ]  Make sure this works in most browsers. (Screw you IE...)
- [ ]  Make it so you can check a single permission to inherit, not all of the permissions in a group.

As I get these things done (or if others want to fork my fork and send me pull requests), I'll check them off (and make a note as to who did what).

### CODE REVIEW

File: Resources/views/admin/partials/permissions.blade.php [hunk 1]
```diff
@@ -7,59 +7,39 @@
...
+                            <input name="permissions[<?php echo "$subPermissionTitle.$permissionAction" ?>]" type="hidden" value="false" />
+                            <input id="<?php echo "$subPermissionTitle.$permissionAction" ?>" name="permissions[<?php echo "$subPermissionTitle.$permissionAction" ?>]" type="checkbox" class="flat-blue" <?php echo array_get($model->permissions, "$subPermissionTitle.$permissionAction", false) === true ? 'checked' : '' ?> value="true" /> {{ ucfirst($permissionAction) }}
+                        </label>
                     </div>
+                    <?php endforeach; ?>
+                </div>
                 <?php endforeach; ?>
+                <div class="clearfix"></div>
+            </div>
+            <?php endforeach; ?>
```
Comment (mikemand) [inline Resources/views/admin/partials/permissions.blade.php]: I'm not sure what's up with this diff. The only changes I made are the same as on `permissions-create.blade.php`.

**Comment by nWidart** [comment]
How is this PR coming along ? Do you need any additional information ?

**Comment by mikemand** [comment]
I had an unexpected project intrude on my time. I'll get back to this as soon as possible.

**Comment by nWidart** [comment]
Okay, no rush was just to get an update. :smile:

**Comment by mikemand** [comment]
Replaced with a much better solution by @nWidart in https://github.com/AsgardCms/User/commit/06d62edc7ca30fb93485d486ab8befc90afa27a8

**Pull request closed without merging**